### PR TITLE
Added Neon Number doc

### LINKED ISSUE #59: Neon Number
## 📚 Documentation

(A clear and concise description of what the issue is.)
Neon Number 
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

(Write your answer here.)
yes

please assign this issue to me

### PR DESCRIPTION
**Pull request by nagasrihith608**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
yes
(Write your answer here.)

### Description
Added Neon Number doc
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
Fixes #59 
(Write your answer here.)

### CODE REVIEW

**Comment by nagasrihith608** [comment]
@HarshCasper please review this pr. Thank you!

**Review by HarshCasper — CHANGES_REQUESTED** [review]
The CI has been broken. Can you please fix it?

**Comment by nagasrihith608** [comment]
@HarshCasper I fixed the broken CI please check now

File: docs/mathematics/neon-number.mdx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+     */
+    public static boolean isNeon(int num) {
+
+        // Accumulator :-> A running total is a sum of numbers that accumulate with each iteration of a loop
+        int n, sum = 0; // assuming sum as accumulator
+
+        n = num * num;
+        while (n != 0) { // running loop till we consider all the digits of the squared number
+            sum += (n % 10); //getting the last digit & adding the last digit to sum
+            n /= 10; // neglecting the last number which we have counted
```
Comment (HarshCasper) [inline docs/mathematics/neon-number.mdx:84]: Don't use comments beside the line of code. Use them above the line of code

File: docs/mathematics/neon-number.mdx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        digit = square_number % 10
+        sum_of_digits += digit
+        square_number = square_number // 10
+    if sum_of_digits == number :
+        print(number,"is a Neon number")
+    else :
+        print(number,"is not a Neon number")
+
+number = int(input("Enter a number : "))
+Neon_number(number)
```
Comment (HarshCasper) [inline docs/mathematics/neon-number.mdx:114]: Enclose in a `if __name__ == '__main__'` block

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Tiny changes. Rest looks good to go off for a peer review.

**Comment by nagasrihith608** [comment]
> Tiny changes. Rest looks good to go off for a peer review.

@HarshCasper just made those changes please have a look :)

**Review by ankitaggarwal23 — APPROVED** [review]

**Comment by HarshCasper** [comment]
Hi @nagasrihith608 

Kindly fix the merge conflict. I will merge it soon after.

**Comment by nagasrihith608** [comment]
@HarshCasper solved the branch conflict

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**